ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/TimeServiceIT.scala

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.client.binding.{Primitive => P}
+import com.daml.ledger.test.semantic.TimeTests._
+import io.grpc.Status
+import java.time.Instant
+
+import scala.concurrent.Future
+
+final class TimeServiceIT extends LedgerTestSuite {
+
+  test(
+    "TSTimeIsStatic",
+    "Time stands still when static time enabled",
+    allocate(NoParties),
+    runConcurrently = false,
+    enabled = _.staticTime,
+    disabledReason = "requires user static time feature",
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    for {
+      initialTime <- ledger.time()
+      _ <- Future { Thread.sleep(100) }
+      laterTime <- ledger.time()
+    } yield {
+      assertEquals("ledger time should stand still", laterTime, initialTime)
+    }
+  })
+
+  test(
+    "TSTimeCanBeAdvanced",
+    "Time can be advanced when static time enabled",
+    allocate(NoParties),
+    runConcurrently = false,
+    enabled = _.staticTime,
+    disabledReason = "requires user static time feature",
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    for {
+      initialTime <- ledger.time()
+      thirtySecLater = initialTime.plusSeconds(30)
+      _ <- ledger.setTime(initialTime, thirtySecLater)
+      laterTime <- ledger.time()
+    } yield {
+      assertEquals("ledger time should stand still", laterTime, thirtySecLater)
+    }
+  })
+
+  test(
+    "TSFailWhenTimeNotAdvanced",
+    "The submission of an exercise before time advancement should fail",
+    allocate(SingleParty),
+    runConcurrently = false,
+    enabled = _.staticTime,
+    disabledReason = "requires user static time feature",
```

Comment:
```suggestion
    disabledReason = "requires ledger static time feature",
```
The static time is about the ledger not the client, no?